Add backlight information (#78)
Eeems-Org/remarkable.guide: src/guide/config/backlight.rst

> ============================
> Changing the Device Backlight
> ============================

Reviewer: ```suggestion
=============================
Changing the Device Backlight
=============================
```

> The ReMarkable Paper Pro has a backlight that can be adjusted to make the screen easier to read in different lighting conditions.

Reviewer: ```suggestion
The reMarkable Paper Pro has a backlight that can be adjusted to make the screen easier to read in different lighting conditions.
```

> Adjusting manually
> ============================

Reviewer: ```suggestion
Adjusting in the UI
===================
```

This section should point to the official documentation on the quick settings: https://support.remarkable.com/s/article/Quick-settings instead of re-documenting how to open them.

Author: The official documentation doesn't provide the numeric values, I think this is quite helpful to have.

Reviewer: Yes, but the numeric values are more important for the programattic section, and not for the UI, and as for the UI we should still be pointing to the official support documentation wherever possible.

Author: I've pointed to the main documentation as well as adding the values

Xochitl provides 5 presets of backlight brightness, which can be accessed by swiping down from the top-right of the screen to open the quick settings menu.

The Xochitl backlight presets are as follows:

#. 260 (12%)
#. 694 (32%)
#. 1040 (50%)
#. 1387 (67%)
#. 1734 (84%)

> Adjusting programmtically
> ============================

Reviewer: ```suggestion
Adjusting programmatically
==========================
```

:raw-html:`<div class="warning">⚠️ Warning. ⚠️<br>`

> Setting the brightness too high may cause the device to overheat and potentially damage the screen.

Reviewer: ```suggestion
Setting the brightness outside of the range you can select in xochitl will make the battery drain faster, and may have unintended side effects that could damage the device.
```
Since this is conjecture, we shouldn't commit to saying what will happen, and instead just say we don't know what will happen.

:raw-html:`</div>`

The backlight can also be adjusted programmatically by writing to the `/sys/class/backlight/backlight/brightness` file.
The value written to this file should be between 0 and 2047, where 0 is completely off and 2047 is the maximum brightness.

.. code-block:: shell

>   echo 2047 > /sys/class/backlight/backlight/brightness

Reviewer: ```suggestion
  echo 1734 > /sys/class/backlight/backlight/brightness
```
Let's avoid having people default to using the max value to test things when we just above say we don't recommend it.

Reviewer 2: echo <num> /sys/class/backlight/rm_frontlight/brightness
